src/app/containers/RecentVideoEpisodes/index.test.jsx

```diff
@@ -28,6 +29,22 @@ describe('RecentAudioEpisodes', () => {
     expect(container.querySelectorAll('li').length).toEqual(3);
   });
 
+  it('should render the list item links', async () => {
+    const { getAllByText } = render(
+      <RecentVideoEpisodesWithContext episodes={recentVideoFixtures} />,
+    );
+
+    const links = getAllByText('BBC Info').map(
+      titleEl => titleEl.closest('a').href,
+    );
+
+    expect(links).toEqual([
+      'http://localhost/afrique/bbc_afrique_tv/tv/w172xc9xq2gllfk',
```

Comment:
this is wrong isn't it? @rebeccamcginn @ryanmccombe 

it should be `/afrique/bbc_afrique_tv/tv_programmes/w172xc9xq2gllfk'`

looking at examples of OD TV 200s in sumologic I'm seeing:

/somali/bbc_somali_tv/tv_programmes/w13xttqt
/hausa/bbc_hausa_tv/tv_programmes/w13xttn0
/burmese/bbc_burmese_tv/tv_programmes/w13xttn3
/hindi/bbc_hindi_tv/tv_programmes/w13xttlw
/kyrgyz/bbc_kyrgyz_tv/tv_programmes/w13xttqx
/gujarati/bbc_gujarati_tv/tv_programmes/w13xttqr
/urdu/bbc_urdu_tv/tv_programmes/w13xttn

Reply:
The episodes have /tv/ and the brands have /tv_programmes/

Reply:
So yes, we can't hardcode 'tv' here https://github.com/bbc/simorgh/pull/8603/files#diff-1af5aa5cd866917e147cc4f1c4e32df72c81843eeb80770873de9cd54e06b19cR68

Reply:
https://www.test.bbc.com/afrique/bbc_afrique_tv/tv/w172xc9xq2gllfk is correct

Reply:
Actually yes, we can hardcode it to 'tv'

Reply:
there won't be a brand page in recent episodes